onnxruntime/test/framework/inference_session_test.cc

```diff
@@ -943,15 +943,10 @@ TEST(InferenceSessionTests, TestOptionalInputs) {
   status = RunOptionalInputTest(true, true, false);
   ASSERT_TRUE(status.IsOK()) << status.ErrorMessage();
 
-  // required, optional and invalid input
-  status = RunOptionalInputTest(true, true, true);
-  ASSERT_FALSE(status.IsOK());
-  EXPECT_THAT(status.ErrorMessage(), testing::HasSubstr("Invalid Feed Input Names: unknown_input"));
-
   // missing required
   status = RunOptionalInputTest(false, true, false);
   ASSERT_FALSE(status.IsOK());
-  EXPECT_THAT(status.ErrorMessage(), testing::HasSubstr("Missing required inputs: required_input"));
+  EXPECT_THAT(status.ErrorMessage(), testing::HasSubstr("Missing required input: required_input"));
```

Comment:
>input [](start = 74, length = 5)

I think there's a C# test that checks for this string that may need updating as well.